framework/src/modules/interoperability/base_cross_chain_update_command.ts

```diff
@@ -174,20 +174,137 @@ export abstract class BaseCrossChainUpdateCommand<
 		}
 	}
 
-	/**
-	 * @param context
-	 * @returns Promise<void>
-	 * @see https://github.com/LiskHQ/lips/blob/main/proposals/lip-0049.md#apply
-	 */
+	private async _beforeCrossChainCommandExecute(
+		context: CrossChainMessageContext,
+		baseEventSnapshotID: number,
+		baseStateSnapshotID: number,
+	): Promise<{ anyError: boolean }> {
```

Comment:
if this just need to return boolean, it would be better to just return `false` when failed and true for success

Reply:
same for below